index.html

```diff
@@ -1,33 +1,60 @@
-    <nav class="navbar navbar-expand-lg navbar-light" style = "background-color: #0083c5;">
-      <div class="container-fluid">
-        <a class="navbar-brand" href="https://acm.mines.edu/">ACM</a> <!-- navbar-brand centers ACM link as the 'brand,' change if you want -->
-        <div class="collapse navbar-collapse" id="navbarNav">
-          <ul class="navbar-nav">
-            <li class="nav-item">
-              <a class="nav-link" aria-current="page" href="pages/blog.html">Blog</a>
-            </li>
-            <li class="nav-item">
-              <a class="nav-link" aria-current="page" href="pages/tutorials.html">Tutorials</a>
-            </li>
-            <li class="nav-item">
-              <a class="nav-link" aria-current="page" href="pages/wiki.html">Wiki</a>
-            </li>
-          </ul>
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <meta name="theme-color" content="#000000">
+  <meta name="description" content="A site for CS@Mines clubs.">
+  <title>Mines ACM</title>
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet"
+    integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+  <link rel="stylesheet" type="text/css" href="/static/index.css" />
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css"
+    integrity="sha512-MV7K8+y+gLIBoVD59lQIYicR65iaqukzvf/nwasF0nqhPay5w/9lJmVM2hMDcnK1OnMGCdVK+iQrJ7lzPJQd1w=="
+    crossorigin="anonymous" referrerpolicy="no-referrer" />
+</head>
+
+<body>
+  <nav class="navbar navbar-expand-lg navbar-light" style="background-color: #0083c5;">
+    <div class="container-fluid">
+      <a class="navbar-brand" href="https://acm.mines.edu/">ACM</a>
+      <!-- navbar-brand centers ACM link as the 'brand,' change if you want -->
+      <div class="collapse navbar-collapse" id="navbarNav">
+        <ul class="navbar-nav">
+          <li class="nav-item">
+            <a class="nav-link" aria-current="page" href="pages/blog.html">Blog</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" aria-current="page" href="pages/tutorials.html">Tutorials</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" aria-current="page" href="pages/wiki.html">Wiki</a>
+          </li>
+        </ul>
       </div>
-    </nav>
-    <h1>Hello world! This site is under construction.</h1>
-    <script src="./static/script.js"></script>
-  </body>
-</html>
+  </nav>
+  <h1>Hello world! This site is under construction.</h1>
+  <script src="./static/script.js"></script>
+  <section class="footer">
+    <div class="social">
+      <a href="#"><i class="fab fa-instagram"></i></a>
+      <a href="#"><i class="fab fa-discord"></i></a>
+      <a href="#"><i class="fab fa-twitter"></i></a>
+      <a href="#"><i class="fab fa-facebook"></i></a>
```

Comment:
I'm not sure what the `fab` class is supposed to do here since it doesn't appear to be used in styling and doesn't seem to have any use in bootstrap, but generally "FAB" stands for Floating Action Button, and is used for a primary action button that floats over content on a page. There should only ever be one FAB per page, and a navlist in the footer doesn't meet that definition. So if that's a bootstrap directive that I just don't know about, you probably shouldn't be using it.

Reply:
It's font awesome icons class that is being imported use a cdn

Reply:
ah, fab = font awesome bold